test/Microsoft.Azure.ServiceBus.UnitTests/API/ApiApprovals.ApproveAzureServiceBus.approved.txt

```diff
@@ -427,7 +453,6 @@ namespace Microsoft.Azure.ServiceBus.Core
     public interface IMessageSender : Microsoft.Azure.ServiceBus.Core.ISenderClient, Microsoft.Azure.ServiceBus.IClientEntity { }
     public interface IReceiverClient : Microsoft.Azure.ServiceBus.IClientEntity
     {
-        string Path { get; }
```

Comment:
Breaking change that will require a new major. Keep that in mind.

Reply:
3.0 is not out of preview yet :)

Reply:
Perfect. Just making sure 😉